helloworld/README.md

```diff
@@ -0,0 +1,33 @@
+# helloworld
+This is a very basic example of how to use
+[Trillian](http://github.com/google/trillian) to implement the operations
+needed for a verifiable log.  It intentionally avoids
+defining any interface between the client and personality, leaving only the
+gRPC interface between the personality and the (Trillian) log.  In a real
```

Comment:
If this is supposed to be The Simplest Thing, we might want to think about how we introduce client / personality / Trillian. If the terms and relationships are well defined elsewhere we can link to that. Some kind of primer would be handy.

Reply:
okay, i've taken a stab at defining clients and personalities.  let me know what you think?